### PR TITLE
Feature/dynamic layout

### PR DESCRIPTION
**Pull request by DemogorGod**
Sup Dev Squad!!!

Just added a dynamic layout for our landing page. 

For now, the links are hidden for all pages except the home page (the Casimir logo)

The dynamic page includes:

- home page with title and side image of the dashboard that will change based on our new dashboard and will include animation
- whitepaper page with an image on the right
- community page with a rotating polygon structure that I will add animations to later
- all pages have dynamic text rendering and sizing
- all pages have dynamic responsiveness with each different size that we have in our layouts 
- Keep in mind that our application's minimum width size is 360px anything below that will be in a scroll bar, but that does not show in the video that I provide 

Let me know what yall think! Thanks! <3

### CODE REVIEW

**Comment by DemogorGod** [comment]
Here is a clip of the the current app

https://user-images.githubusercontent.com/63758598/197673114-6178342f-f9af-4d01-a5f0-d86e332bfa0d.mov

File: apps/landing/src/pages/home/Home.vue [hunk 1]
```diff
@@ -30,18 +29,14 @@ import NavBar from '../../components/nav/NavBar.vue'
...
-      <div class="px-margins mt-[100px]">
-        <h2 class="font-regular text-grey_5">
-          Simple self custody for all of your digital assets
-        </h2>
-        <h5 class="text-grey_2 mt-[40px]">
+  <div class="grid grid-cols-6">
+    <div class="bg-black col-span-4 csm-md:col-span-6 h-routerview">
+      <div class="px-margins mt-[20px]">
+        <h5 class="text-grey_2">
           Non-custodial staking and asset managment platform
```
Comment (shanejearley) [inline apps/landing/src/pages/home/Home.vue:36]: *management

File: apps/landing/src/components/nav/NavBar.vue [hunk 2]
```diff
@@ -1,39 +1,94 @@
...
-      class="my-auto"
-    >
-      <h6
-        class="nav-links"
-        :class="$route.fullPath.split('/')[1] === 'community'? 'nav-links-selected' : ''"
+        <h3 class="font-medium text-white mb-[25px] nav-xsm:text-[24px] nav-xsm:leading-[30px] nav-xsm:font-semibold">
+          Why We Are building Casimir
+        </h3>
+        <h5 class="text-blue_1 font-light nav-xsm:text-[16px] nav-xsm:leading-[20px]">
+          Written by: Nathan Miller
```
Comment (shanejearley) [inline apps/landing/src/components/nav/NavBar.vue:56]: By Nathan Miller

File: apps/landing/src/components/nav/NavBar.vue [hunk 3]
```diff
@@ -1,39 +1,94 @@
...
-    <!-- <RouterLink 
-      to="/why-casimir"
-      class="my-auto"
-    >
-      <h6
-        class="nav-links"
-        :class="$route.fullPath.split('/')[1] === 'why-casimir'? 'nav-links-selected' : ''"
+        <h2 class="font-regular text-grey_5 home-nav-dynamic-text">
+          <!-- sm:text-[36px] truncate -->
+          Simple self custody for all of your digital assets
```
Comment (shanejearley) [inline apps/landing/src/components/nav/NavBar.vue:45]: *self-custody

**Review by shanejearley — CHANGES_REQUESTED** [review]
Looks good. I opened the Slack deployment preview on my phone and the layout fit very nicely.

**Comment by shanejearley** [comment]
Do you mind also deleting the users folder (`services/users`) before your next commit? Apologies, this slipped in there because of file name changes a few weeks ago.

**Comment by DemogorGod** [comment]
@shanejearley Fixed spelling issues and deleted `/services/users' as well

**Review by shanejearley — APPROVED** [review]
Looks good, nice changes. I will push a commit to fix the merge conflicts and then you can merge it.

**Pull request merged**